src/openassetio-python/tests/cmodule/gil/test_manager_gil.py

```diff
@@ -56,8 +56,8 @@ def test_all_methods_covered(self, find_unimplemented_test_cases):
             for method in unimplemented:
                 print(
                     f"""
-    def test_{method}(self, a_threaded_manager, a_context, a_host_session):
-        a_threaded_manager.{method}(a_context, a_host_session)
```

Comment:
[Question] Why the removal of `host_session`. This is searching for unimplemented manager methods isn't it, shouldn't `host_session` remain?

Reply:
It was a copy-paste error when it was introduced - no `Manager` method takes a `HostSession` instance.  I must have copy-pasted it from `ManagerInterface`.

Reply:
Right, i see. Thing about `Manager` and `ManagerInterface` differently from the external perspective still bites me.